### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by athy9193**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+
+Hunting for rentals can be an exhausting and frustrating experience in Canada, but this process can be made easy with a simple installation of our package. This Python package intends to facilitate the house hunting process by scraping the listing information from Craigslist and organizing the extracted data for the user. Instead of having to manually go on the website to catch up with individual new listings, the user will be updated through email with new results as per their selection criteria. 
+
+
+## Functions
+
+| Function Name | Input | Output | Description |
+|-----------|------------|---------------|------------------|
+| scraper | url | tibble | Scrape data from rental websites into a csv file.|
+| data_cleaner | tibble | tibble | Clean the extracted data |
```
Comment (JuntingHe) [inline README.md:19]: Hi Alex. I actually document the input and output as a data.frame.

**Review by JuntingHe — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+| data_filter | tibble, min_price, max_price, sqrt_ft, num_bathroom, num_bedroom, neighbourhood | tibble | Filter the cleaned data set based on user inputs|
+| send_email | tibble, email address | csv file | Send the organized listing information to user email |
```
Comment (elabandari) [inline README.md:21]:
```suggestion
| send_email | tibble, email address | None | Send the organized listing information to user email |
```

File: README.md (hunk 1 above)
Comment (athy9193) [reply README.md:19]: This is R so I think it is tibble :)

**Review by elabandari — APPROVED** [review]

**Review by JuntingHe — APPROVED** [review]
Looks nice!!!

**Pull request merged**